tests/prophet/test_prophet_model_export.py

```diff
@@ -202,49 +202,48 @@ def test_model_load_from_remote_uri_succeeds(prophet_model, model_path, mock_s3_
     )
 
 
-def test_model_log(prophet_model):
-    with TempDir(chdr=True, remove_on_exit=True) as tmp:
-        for should_start_run in [False, True]:
-            try:
-                if should_start_run:
-                    mlflow.start_run()
-                artifact_path = "prophet"
-                conda_env = os.path.join(tmp.path(), "conda_env.yaml")
-                _mlflow_conda_env(conda_env, additional_pip_deps=["pystan", "prophet"])
-
-                model_info = mlflow.prophet.log_model(
-                    pr_model=prophet_model.model, artifact_path=artifact_path, conda_env=conda_env
-                )
-                model_uri = f"runs:/{mlflow.active_run().info.run_id}/{artifact_path}"
-                assert model_info.model_uri == model_uri
-                reloaded_prophet_model = mlflow.prophet.load_model(model_uri=model_uri)
-
-                np.testing.assert_array_equal(
-                    generate_forecast(prophet_model.model, FORECAST_HORIZON),
-                    generate_forecast(reloaded_prophet_model, FORECAST_HORIZON),
-                )
-
-                model_path = _download_artifact_from_uri(artifact_uri=model_uri)
-                model_config = Model.load(os.path.join(model_path, "MLmodel"))
-                assert pyfunc.FLAVOR_NAME in model_config.flavors
-                assert pyfunc.ENV in model_config.flavors[pyfunc.FLAVOR_NAME]
-                env_path = model_config.flavors[pyfunc.FLAVOR_NAME][pyfunc.ENV]
-                assert os.path.exists(os.path.join(model_path, env_path))
-
-            finally:
-                mlflow.end_run()
-
-
-def test_log_model_calls_register_model(prophet_model):
+def test_prophet_log_model(prophet_model, tmpdir):
+    for should_start_run in [False, True]:
+        try:
+            if should_start_run:
+                mlflow.start_run()
+            artifact_path = "prophet"
+            conda_env = pathlib.Path(tmpdir).joinpath("conda_env.yaml")
```

Comment:
```suggestion
            conda_env = tmp_path.joinpath("conda_env.yaml")
```

Can we replace `tmpdir` with `tmp_path`?